react/training-project/src/app/components/Square/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import style from './styles.scss';
+
+const Square = props => (
```

Comment:
Despite you're going to turn it into a class component, try to avoid arrow functions to generate components. Use:
```
function MyComponent({ value }) {
...
}
```